CONTRIBUTOR_LICENSE_AGREEMENT.md

```diff
@@ -0,0 +1,13 @@
+ - Grant of Copyright License. Subject to the terms and conditions of this Agreement, You hereby grant to Code42 and to recipients of software distributed by Code42 a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable copyright license to reproduce, prepare derivative works of, publicly display, publicly perform, sublicense, and distribute Your Contributions and such derivative works.
+ - Grant of Patent License. Subject to the terms and conditions of this Agreement, You hereby grant to Code42 and to recipients of software distributed by Code42 a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable (except as stated in this section) patent license to make, have made, use, offer to sell, sell, import, and otherwise transfer the Work, where such license applies only to those patent claims licensable by You that are necessarily infringed by Your Contribution(s) alone or by combination of Your Contribution(s) with the Work to which such Contribution(s) was submitted. If any entity institutes patent litigation against You or any other entity (including a cross-claim or counterclaim in a lawsuit) alleging that your Contribution, or the Work to which you have contributed, constitutes direct or contributory patent infringement, then any patent licenses granted to that entity under this Agreement for that Contribution or Work shall terminate as of the date such litigation is filed.
+ - You represent that you are legally entitled to grant the above license. If your employer(s) has rights to intellectual property that you create that includes your Contributions, you represent that you have received permission to make Contributions on behalf of that employer, that you will have received permission from your current and future employers for all future Contributions, that your applicable employer has waived such rights for all of your current and future Contributions to Code42, or that your employer has executed a separate Corporate Contributor License Agreement with Code42.
+ - You represent that each of Your Contributions is Your original creation (see section 7 for submissions on behalf of others). You represent that Your Contribution submissions include complete details of any third-party license or other restriction (including, but not limited to, related patents and trademarks) of which you are personally aware and which are associated with any part of Your Contributions.
```

Comment:
can't find `section 7`.

Reply:
I reformatted the markdown with section numbers and the agreement itself is now hosted in it's own repo: https://code42.github.io/code42-cla/Code42_Individual_Contributor_License_Agreement